### PR TITLE
Feat: Liquidity Providers 🐋 

### PR DESCRIPTION
**Pull request by sofinico**
Task completed CoinFabrik/alex-qa-internal#30, ready for review.

Implemented [Template v0.0.1](https://github.com/CoinFabrik/alex-qa-internal/wiki/Templates-Soft#alex-lab-features), so it might be subjected to changes.

### CODE REVIEW

File: product-features/liquidity-providers/how-to.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+description: >-
+  In these two guides, you'll find the required steps to provide liquidity and
+  to withdraw liquidity from ALEX DEX pools.
+---
+
+# How to add/remove liquidity
+
+When **adding liquidity**, you will deposit an equivalent value of both tokens into the pool. In return, you’ll receive LP tokens, which represent your share of that specific liquidity pool.
+
+When **removing liquidity**, you will transfer your LP tokens back to the ALEX smart contract and receive your original funds plus any fees accrued while holding those LP tokens.
```
Comment (sofinico) [inline product-features/liquidity-providers/how-to.md]: @XianPaz I am not sure if these clarifications should go here at the beginning or below each "Adding/Removing Liquidity" subtitle. Plz let me know your thoughts.

Comment (XianPaz) [reply product-features/liquidity-providers/how-to.md]: Here is OK

**Review by XianPaz — APPROVED** [review]

**Review by tenuki — CHANGES_REQUESTED** [review]
En product-features/liquidity-providers/faqs.md: "When a user adds liquidity to a pool, they receive LP tokens as proof of ownership, entitling them to a proportional share of the pooled assets and any fees generated by trades (swaps) in that pool."  ->  When a user adds => When an user adds..     pero en general queda mejor asi: "When users add liquidity to a pool, they receive LP tokens as proof of ownership. These tokens entitle them to a proportional share of the pooled assets and a portion of the fees generated by trades (swaps) within the pool."

**Review by tenuki — APPROVED** [review]

**Comment by sofinico** [comment]
Cerrado porque el PR se hace directamente `alexgo-io/alexlab-doc:main`.

**Pull request closed without merging**